src/views/Registration/RegistrationDefineBusiness.vue

```diff
@@ -9,7 +9,19 @@
       <v-card flat class="mt-5">
         <NameRequestInfo />
 
-        <!-- FUTURE: Business Type goes here -->
+        <template v-if="isTypePartnership">
+          <v-divider class="mx-6" />
+
+          <!-- Business Type -->
+          <header id="business-type-header" />
+          <BusinessTypeGP
+            class="py-8 px-6"
+            :class="{ 'invalid-section': getShowErrors && !businessTypeGPValid }"
+            :showErrors="getShowErrors"
```

Comment:
You don't need to pass this prop since your component doesn't use it.

Reply:
Okay. Thanks. I will remove it.